docs/source/execute_api.rst

```diff
@@ -0,0 +1,143 @@
+Executing notebooks
+===================
+
+.. module:: nbconvert.preprocessors
+
+In this section we show how to execute a ``.ipynb`` notebook
```

Comment:
@takluyver Good point. It's been done both ways throughout the docs. Since we do use ``.ipynb`` throughout the docs instead of ``ipynb`` file, I agree with you. Let's leave this as "a" ``.ipynb`` file.